website_openstreetmap/views/widget.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <!-- widget structure inferred from image floating widget it is placed in content and can be inside a text -->
+        <template id="openstreetmap_widget" inherit_id="website.snippets" name="openstreetmap">
+            <xpath expr="//div[@id='snippet_content']" position="inside"> <!--this decides the category the snippet belongs to -->
+                <div name="openstreetmap_floating" class="o_block_image_float">
```

Comment:
The standard way to define a snippet is like this:

`<t t-snippet="website.s_well" t-thumbnail="/website/static/src/img/blocks/block_well.png"/>`

`<template id="s_well" name="Well"></template>`